backend/codesirius/problems/views/tag.py

```
@@ -51,48 +51,63 @@
         pk = kwargs.get("pk")
         logger.info(f"Partially updating tag with ID: {pk}")
-        response = super().partial_update(request, *args, **kwargs)
-        logger.info(f"Tag with ID: {pk} partially updated successfully")
-        return CodesiriusAPIResponse(
-            data={"id": response.data.get("id")},
-            status_code=status.HTTP_200_OK,
-            message="Tag updated",
-        )
+        try:
+            response = super().partial_update(request, *args, **kwargs)
+            logger.info(f"Tag with ID: {pk} partially updated successfully")
```

Ruff (0.8.2)

Logging statement uses f-string

(G004)